code/datums/materials/_material.dm

```diff
@@ -12,6 +12,8 @@ Simple datum which is instanced once per type and is used for every object of sa
 	var/color
 	///Base alpha of the material, is used for greyscale icons.
 	var/alpha
+	///Bitflags that influence how SSmaterials handles this material.
+	var/material_flags = MATERIAL_INIT_MAPLOAD
```

Comment:
We now have a similarly named var on atoms which handles how the atom treats materials, I can imagine that can confuse people.

Reply:
This one I can solve easily enough.

Reply:
<3

Reply:
Is `init_flags` a good enough name?

Reply:
yeah seems fine :D We can always change it if we want to add different types of flags :)